### ISSUE TITLE
[BUG] unable to install data

### ISSUE THREAD

**Issue by RwanMahmoud** [CAMeL-Lab/camel_tools#83]
I am trying to work with CAMEL, the installation is fine, however I get the following error when I try to download the data:
An error occured while extracting downloaded data. I tried both full and light data.
 - OS macOS
 - Python 3.7.4

**Comment by owo** [comment]
We are aware of this issue and we will provide a fix soon.

**Comment by owo** [comment]
We have pushed a fix for the issue.
Please upgrade camel-tools by running `pip install --upgrade camel-tools`.
The data installation command is slightly different now so you have to run either `camel_data -i all` or `camel_data -i light`.

**Comment by lancioni** [comment]
I tried it (with Python 3.10 under Windows 10), but the command is not understood:

Usage:
    camel_data [-d <DIR> | --data-dir=<DIR>] <PACKAGE>
    camel_data (-l | --list)
    camel_data (-v | --version)
    camel_data (-h | --help)

This is why I resorted to the old way...

**Comment by owo** [comment]
Sadly Python 3.10 is not supported at the moment as some of our dependencies don't support it yet.
We only support Python 3.7-3.9 for now.

As for the fact you are using Windows, for some reason, pip didn't make the latest release available on Windows so the current instructions won't work. We are currently working to fix this problem for the upcoming release (which should be very soon).

I'll update you here when we post the next release.

**Comment by lancioni** [comment]
I tried with Python 3.9, but there is an issue compiling camel-kenlm under Windows. The same issue did already exist for the original kenlm, which I managed to compile under Linux, but not Windows. I guess in my previous configuration (Python 3.6) the Windows version did not include kenlm, did it?

  Running setup.py install for camel-kenlm ... error
  error: subprocess-exited-with-error

  × Running setup.py install for camel-kenlm did not run successfully.
  │ exit code: 1
  ╰─> [324 lines of output]
